src/main/java/com/iota/iri/conf/BaseIotaConfig.java

```diff
@@ -600,9 +601,10 @@ protected void setBelowMaxDepthTransactionLimit(int maxAnalyzedTransactions) {
     }
 
     public interface Defaults {
-        //API
+		//API
         int API_PORT = 14265;
         String API_HOST = "localhost";
+        List<String> REMOTE_LIMIT_API = IotaUtils.createImmutableList("addNeighbors", "getNeighbors", "removeNeighbors", "attachToTangle", "interruptAttachingToTangle");
```

Comment:
Can you set your IDE to have a column width of 120.
I will try to add more checkstyle rules real soon. I know style comments are annoying. A computer should give them and not a human.